templates/zerver/emails/missed_message.html

```diff
@@ -29,6 +29,8 @@
     {% trans %}You are receiving this because you were personally mentioned.{% endtrans %}<br />
     {% elif mentioned_user_group_name %}
     {% trans %}You are receiving this because @{{ mentioned_user_group_name }} was mentioned.{% endtrans %}<br />
+    {% elif topic_wildcard_mentioned %}
+    {% trans %}You are receiving this because everyone was mentioned in #{{ stream_name }} > {{ topic_name }}.{% endtrans %}<br />
```

Comment:
This should be discussed in #design, but I was thinking of something more like "because @topic was mentioned in #{{ stream_name }} > {{ topic_name }}.` or "because all participants were mentioned in #{{ stream_name }} > {{ topic_name }}.`

Reply:
I made the changes after [discussion in CZO.](https://chat.zulip.org/#narrow/stream/101-design/topic/notification.20emails/near/1588768)

> You are receiving this because all topic participants were mentioned in #{{ stream_name }} > {{ topic_name }}.